cumulusci/core/flowrunner.py

```diff
@@ -160,19 +164,48 @@ def __init__(self, model):
             def post_task(self, step, result):
                 # do something to record state on self.model
 
-    (An instance of the custom FlowCallback class would be passed to FlowCoordinator.)
+    Once a subclass is defined, you can instantiate it, and
+    pass it as the value for the 'callbacks' keyword argument
+    when instantiating a FlowCoordinator.
+
+    Example running a flow with custom callbacks:
+
+        custom_callbacks = CustomFlowCallbacks(model_instance)
+        flow_coordinator = FlowCoordinator(
+            project_config,
+            flow_config,
+            name=flow_name,
+            options=options,
+            callbacks=custom_callbacks,
+        )
+        flow_coordinator.run(org_config)
+
+
     """
 
     def pre_flow(self, coordinator):
+        """This is passed an instance of FlowCoordinator,
+        that pertains to the flow which is about to run."""
         pass
 
     def post_flow(self, coordinator):
+        """This is passed an instance of FlowCoordinator,
+        that pertains to the flow just finished running."""
```

Comment:
We should clarify that this runs whether or not the flow completed successfully.